docs/source/convert.rst

```diff
@@ -63,8 +63,7 @@ netCDF file named ``FILENAME.nc`` saved to the directory path
 
    - Prior to version 0.5.0, conversion was carried out through the
      `"Convert" interface <https://echopype.readthedocs.io/en/v0.4.1/usage.html#conversion-operation>`_.
-     This interface is still available but will be deprecated in
-     a future version.
+     This interface is deprecated in version 0.6.0.
```

Comment:
Come to think of it, we previously should've said that the interface was *deprecated* [(ie, discouraged, no longer recommended or actively developed, but still available)](https://en.wikipedia.org/wiki/Deprecation) and would be *removed* in a future version. What we're doing now is removing it, the next step after deprecation. So, replace with: "This deprecated interface was removed in version 0.6.0."

Alternatively, at this stage how about just removing all references to it from the docs? That would be true also for the references to raw2nc and raw2zarr, below, though I see that those methods are not being removed in this PR (I'll comment about that elsewhere).

Reply:
I removed references to the old API completely from the docs. I agree that is a better approach at this stage.